models/concerns/searchable.rb

```diff
@@ -0,0 +1,40 @@
+module Searchable
+  extend ActiveSupport::Concern
+
+  included do
+    include Elasticsearch::Model
+
+    # We specify our own callbacks, instead of using Elasticsearch::Model::Callbacks, so that we can disable
```

Comment:
@clintonb: Is the whole concept of `CommentService.search_enabled?` for tests only?  I wish there were another way to handle this that doesn't affect the actual code.